add systemd services for configuration after start (#980)
crc-org/snc: notes-self-sufficient.md

# Self sufficient bundles

> Since release 4.19.0 of OpenShift Local, the bundles generated by `snc` contains additional systemd services to provision the cluster and removes the need of

Reviewer: ```suggestion
Since release 4.19.0 of OpenShift Local, the bundles generated by `snc` contain additional systemd services to provision the cluster and remove the need for
```

> an outside entity for provision the cluster, although an outside process needs to create some files on pre-defined locations inside the VM for the  systemd

Reviewer: ```suggestion
an outside entity to provision the cluster, although an outside process needs to create some files on pre-defined locations inside the VM for the  systemd
```

services to do their work.

> ## The following table lists the systemd services and the location of files they need to provision the cluster, users of SNC needs to create those files

Reviewer: ```suggestion
## The following table lists the systemd services and the location of files they need to provision the cluster, users of SNC need to create those files
```

|    crc-pullsecret     |               both               |         /opt/crc/pull-secret         |         none         |
|      crc-dnsmasq      |               both               |                 none                 |         none         |
| crc-routes-controller |               both               |                 none                 |         none         |
|    ocp-cluster-ca     |               ocp                |        /opt/crc/custom-ca.crt        |     CRC_CLOUD=1      |
|     ocp-clusterid     |               ocp                |                 none                 |         none         |
|   ocp-custom-domain   |               ocp                |                 none                 |     CRC_CLOUD=1      |
|      ocp-growfs       |               ocp                |                 none                 |         none         |
|   ocp-userpasswords   |               ocp                | /opt/crc/pass_{kubeadmin, developer} |         none         |

> [!NOTE]
> "Marker env variable" is set using an env file, if the required env variable is not set then unit is skipped

> > some units are ran only when CRC_CLOUD=1 is set, these are only needed when using the bundles with crc-cloud

Reviewer: ```suggestion
> some units are run only when CRC_CLOUD=1 is set, these are only needed when using the bundles with crc-cloud
```
